aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -23,6 +23,32 @@
 from ..error import WalletKeyMissingError
 
 
+ACAPY_LIFECYCLE_CONFIG_FLAG_MAP = {
```

Comment:
I wonder if we should have another map too, one that better matches config.yml ( so`auto_accept_invites`: `debug.auto_accept_invites).  I am OK with not having it and staying with the ENV_VAR format but could see the case where they are migrating an existing config.yml file contents.